### PR TITLE
hello k12 & hirschhof

### PR DESCRIPTION
**Pull request by Noki**
added k12-haus4 and hirschhof - do not merge yet, still some issues with ethernet / vlans

### CODE REVIEW

**Comment by Noki** [comment]
ethernet / vlan issues solved. remaining problem is unfunctional mehs on 5GHz radio on TPlink Archer C5 v1

**Comment by Noki** [comment]
Alle Probleme beseitigt und fertig zum Merge.

**Comment by PolynomialDivision** [comment]
lgtm

File: group_vars/location_k12/networks.yml [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+# MGMT
+  - vid: 42
+    role: mgmt
+    prefix: 10.31.158.192/26
+    gateway: 1
+    dns: 1
+    ipv6_subprefix: 1
+    assignments:
+      #10.31.158.193/32
```
Comment (robertfoss) [inline group_vars/location_k12/networks.yml:115]: Is a POE switch used? If so maybe it should have a static mgmt ip. If not, please disregard this comment.

Comment (Noki) [reply group_vars/location_k12/networks.yml:115]: No fancy stuff like POE switches here...

File: group_vars/location_k12_haus4/networks.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      #10.31.157.178/32
+      k12-haus4-hirschhof: 2
+
+# AP-id, wifi-channel, bandwidth, txpower
+location__channel_assignments_11a_standard__to_merge:
+  k12-haus4-core: 44-40
+
+location__channel_assignments_11g_standard__to_merge:
+  k12-haus4-core: 13-20
+  k12-haus4-hirschhof: 13-20
```
[the remark this answers is not in the document]
Comment (Noki) [reply group_vars/location_k12_haus4/networks.yml:74]: Yes, intentionally but indeed suboptimal. The core-router has a penalty on the 2 GHz radio, but we want to keep both on channel 13 for now so other people in the area could connect. I hope that we can switch the k12<->hirschhof connection to 5 GHz soon to resolve this.

**Pull request merged**